docker-compose.yml

```diff
@@ -22,12 +22,14 @@ services:
       WEBPACK_MANIFEST_HOST: "webpack"
       WEBPACK_MANIFEST_PORT: "3808"
       WEBPACK_PORT: "3808"
+      MAIN_HOST: "localhost:3000"
       DB_USERNAME: 'postgres'
       DB_PASSWORD: ''
       DB_HOST: 'postgres'
       REDISCLOUD_URL: 'redis://redis:6379'
     volumes:
       - .:/app
+      - node_modules:/app/node_modules
```

Comment:
These two lines where you add node_modules as a volume as unnecessary. Since we already have a volume pointing to the app directory.